genai-perf/genai_perf/inputs/converters/base_converter.py

```diff
@@ -60,3 +74,10 @@ def _construct_text_payload(self, input_data: Dict) -> str:
         """
         contents = [v for k, v in input_data.items() if k in self._CONTENT_NAMES]
         return " ".join(contents)
+
+    def _construct_texts_payload(self, input_data: List) -> List:
```

Comment:
can we change the function name to something like `_construct_batched_text_payload` for better distinguish from the function above?

Reply:
Sure, done.